packages/playback/src/lib/pipeline-loaders/hls-pipeline-loader.ts

```diff
@@ -18,6 +17,10 @@ export class HlsPipelineLoader implements IPipelineLoader {
     HlsPipelineLoader.hlsParserFactory_ = parser;
   }
 
+  public static getHlsParser(): typeof ChunkPlaylistParser | null {
+    return HlsPipelineLoader.hlsParserFactory_;
+  }
```

Comment:
Can we rename getters and setter as `getHlsParserFactory` and `setHlsParserFactory`?